Sources/HAP/Controllers/PairSetupController.swift

```diff
@@ -88,18 +127,22 @@ class PairSetupController {
 
         guard let plaintext = try? ChaCha20Poly1305.decrypt(cipher: encryptedData,
                                                             nonce: "PS-Msg05".data(using: .utf8)!,
-                                                            key: encryptionKey) else {
-                                                                throw Error.couldNotDecryptMessage
+                                                            key: encryptionKey)
+            else {
+            device.changePairingState(.notPaired)
+            throw Error.couldNotDecryptMessage
```

Comment:
These two lines could use an extra level of indentation for readability.

Reply:
Got it